commercial/app/model/merchandise/events/Eventbrite.scala

```diff
@@ -199,21 +122,28 @@ object Eventbrite extends ExecutionContexts with Logging {
     }
   }
 
-  trait EBTicketHandler {
-    def tickets: Seq[EBTicket]
+  trait TicketHandler {
+    def tickets: Seq[Ticket]
+
+    lazy val displayPriceRange = {
```

Comment:
Okay - I can't see/find a reason that we need this to be lazy, so I'm taking it out.